{
    "type": "bundle",
    "id": "bundle--76693ca0-4fc3-497f-b1bd-b8742a9ed467",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--bba611df-109e-491e-bdee-70e4451d4d42",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-02-22T00:20:38.952818Z",
            "modified": "2025-02-22T00:20:38.952818Z",
            "name": "CVE-2025-26013",
            "description": "An issue in Loggrove v.1.0 allows a remote attacker to obtain sensitive information via the read.py component.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-26013"
                }
            ]
        }
    ]
}